### PR TITLE
Profile Type Icon In Chat + Typo Fix

### PR DESCRIPTION
**Pull request by CattoFace**
Added a feature to display the profile type of a player in chat (ironman/bingo).
The feature requires the other player to be loaded for the icon to appear and there is a cache of recent players to keep the icon in case one goes so far they unload or they leave the lobby.

Also fixed a string typo in the item dropping feature that caused an unwanted string to print instead of a localized one,

### CODE REVIEW

File: src/main/resources/lang/he_IL.json [hunk 1]
```diff
@@ -219,7 +219,8 @@
     "showSkyblockItemId": "Show Skyblock Item ID on Hover",
     "showFeatureNamesOnHover": "Show Feature Names on Hover",
     "resetSalvagedEssencesAfterLeavingMenu": "Reset Salvaged Essence Counter After Closing Menu",
-    "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys"
+    "changeDungeonMapZoomWithKeyboard": "Adjust zoom with +/- keys",
+    "showProfileTypeInChat": "הוסף סוג פרופיל לצאט"
```
Comment (ILikePlayingGames) [inline src/main/resources/lang/he_IL.json:223]: Please revert this change and translate the string on Crowdin after the PR is merged. https://crowdin.com/project/skyblockaddons

**Review by ILikePlayingGames — CHANGES_REQUESTED** [review]

**Comment by ILikePlayingGames** [comment]
Thank you for the PR!

File: src/main/resources/lang/he_IL.json (hunk 1 above)
Comment (CattoFace) [reply src/main/resources/lang/he_IL.json]: Done

**Review by ILikePlayingGames — APPROVED** [review]

**Review by ILikePlayingGames — APPROVED** [review]
